legend-engine-xt-relationalStore-test-server/pom.xml

```diff
@@ -0,0 +1,255 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I see what you're doing here .. moving db specific test code from the engine server module to a test module specific to db tests. 

Is this pattern only for relational connections ? For e.g will there be a serviceStore-test-server ?

Reply:
Yes, I see us creating serviceStore-test-server and so on in future (if it helps in integration testing for those stores)